### PR TITLE
feat(plugin): FixYoutubeEmbeds - fix UMG blocked embeds

### PR DESCRIPTION
**Pull request by CoolElectronics**
Allows embedded youtube videos to be played when they're usually blocked.

I *believe* youtube uses window.location.ancestorOrigins to detect this, and it's unreasonably hard to overwrite it. Instead I just reload the frame which clears ancestorOrigins and shouldn't break randomly if the frontend updates.

### CODE REVIEW

**Comment by CoolElectronics** [comment]
umg

File: src/plugins/fixYoutubeEmbeds.desktop/index.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+ * Copyright (c) 2023 Vendicated and contributors
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ */
+
+import { Devs } from "@utils/constants";
+import definePlugin from "@utils/types";
+
+export default definePlugin({
+    name: "FixYoutubeEmbeds",
+    description: "Fixes youtube embeds from being blocked",
```
Comment (Vendicated) [inline src/plugins/fixYoutubeEmbeds.desktop/index.ts:12]: i think smth like that would be better
```suggestion
    description: "Bypasses youtube videos being blocked from display on Discord (for example by UMG)",
```

**Comment by Vendicated** [comment]
please add a README.md to your plugin that explains in a bit more detail what this does and maybe show a before and after screenshot

**Comment by CoolElectronics** [comment]
Added!

**Comment by Vendicated** [comment]
thank you!

**Pull request merged**